### ISSUE TITLE
Segmentation fault for FASTQ (merged reads from NovaSeq)

### ISSUE THREAD

**Issue by vmikk** [smithlabcode/falco#12]
Hello!

I'm getting error while trying to run `falco` on merged reads (FASTQ, NovaSeq):
```
[Tue Dec 22 15:05:11 2020] Started reading file smp.fq.gz
[Tue Dec 22 15:05:11 2020] reading file as gzipped fastq format
Segmentation fault (core dumped)
```
Merging was done with BBMap tools.
FASTQ looks normal, no validation errors found with `fq lint` (with [fqlib](https://github.com/stjude/fqlib)).
`FastQC` accepts this file without errors.

`falco` v.0.2.1, from bioconda repository.
Please find the example file in the attachment ([smp.fq.gz](https://github.com/smithlabcode/falco/files/5730026/smp.fq.gz))

With kind regards,
Vladimir

**Comment by guilhermesena1** [comment]
Hi Vladimir,

Thank you for reporting the error and providing the accompanying data to reproduce. I believe this issue resembles Issue #9 , which was fixed in the most recent versions (0.2.2 and 0.2.3). However, and this is completely my fault, falco on conda is out of data and still points to 0.2.1. I have submitted a pull request to bioconda to update it (I'll post here when the PR is accepted), after which I believe the problem should be solved by running `conda update falco` . In the mean time, if the problem is urgent, you can download version 0.2.3 from the releases page in the github repo. I'm sorry once again that you ran into this segfault.

Best,
Guilherme

**Comment by vmikk** [comment]
Hello Guilherme!
I'm sorry, I haven't noticed that this issue was already solved in the new version of `falco`. I will use `0.2.3` from the GitHub releases.

Thank you so much for your prompt reply and the nice software!
With kind regards,
Vladimir
